exercises/isbn-verifier/isbn_verifier_test.exs

```diff
@@ -0,0 +1,74 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("isbn_verifier.exs", __DIR__)
+end
+
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
+
+defmodule ISBNVerifierTest do
+  use ExUnit.Case
+
```

Comment:
Could you please leave the `@tag :pending` here, but commented out? At this point it's mostly for consistency's sake, but it's also helpful for folks to see sometimes.

Reply:
Got it :)